src/quo2/foundations/colors_test.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.foundations.colors-test
+  (:require [cljs.test :refer-macros [deftest testing is]]
+            [quo2.foundations.colors :as colors]))
+
+(deftest test-color-resolver
```

Comment:
The idiomatic naming of the test var is usually `<function-name>-test`, so `custom-color-test`, or in case you want to separate the `deftest`s, `custom-color-resolver-with-opacity-test` and `custom-color-resolver-with-default-opacity-test`.

Anything that helps the reader quickly understand which function is under test will be fine imo. Here I had to scan the assertions to be sure we were testing only `custom-color`.